test/routes.js

```diff
@@ -849,6 +850,103 @@ test('Custom error handler', async (t) => {
   t.strictEqual(res.statusCode, 403)
 })
 
+test('Graphql Error handling (without data)', async (t) => {
+  const app = Fastify()
+  const schema = `
+    type Query {
+      add(x: Int, y: Int): Int
+    }
+  `
+
+  const resolvers = {
+    add: async ({ x, y }) => { throw new GraphQLError('Graphql Error in resolver') }
+  }
+
+  function errorHandler (error, req, reply) {
+    reply.code(200)
+    reply.send({ errors: error.errors })
+  }
```

Comment:
Is this needed to test this feature?

Reply:
for now, `errors` is an object

firstly, we need to return status 200
secondary, we need to return errors as an array

Reply:
Can you change the code in routes so that this block is not needed?